packages/relay/src/lib/eth.ts

```diff
@@ -1797,30 +1201,6 @@ export class EthImpl implements Eth {
     return input.startsWith(CommonService.emptyHex) ? input.substring(2) : input;
```

Comment:
`prune0x` and `getTransactionCountFromBlockResponse` are no longer used here and can be removed